### PR TITLE
12.0 afc 1337

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

File: outplacement_final_report/models/outplacement.py [hunk 1]
```diff
@@ -32,31 +34,26 @@ class OutplacementGoal(models.Model):
...
-        ('market demand', 'Efterfrågan på arbetsmarknaden'),
-        ('complementing education', 'Kompletterar nuvarande utbildning'),
-        ('complementing experience', 'kompletterar tidigare erfarenheter'),
-        ('other', 'Other')
-        ], string="Motivation")
-    # should only be used if motivation is "other"
-    free_text = fields.Char(string="Free text")
     step_ids = fields.Many2many(
         comodel_name="outplacement.goal.step", string="Step")
+    job_description = fields.Char(string="Job description")
```
Comment (nils-vertel) [inline outplacement_final_report/models/outplacement.py:39]: Trailing comas in field definitions makes Odoo sad

File: outplacement_final_report/models/outplacement.py [hunk 2]
```diff
@@ -32,31 +34,26 @@ class OutplacementGoal(models.Model):
...
-    free_text = fields.Char(string="Free text")
     step_ids = fields.Many2many(
         comodel_name="outplacement.goal.step", string="Step")
+    job_description = fields.Char(string="Job description") 
+    matches_interest = fields.Boolean(string='Matches interest'),
+    matches_ability = fields.Boolean(string='Matches ability'),
+    market_demand = fields.Boolean(string='Market demand'),
+    complementing_education = fields.Boolean(string='Complementing education'),
+    complementing_experience = fields.Boolean(string='Complementing experience'),
+    other_motivation = fields.Boolean(string='Other')
```
Comment (nils-vertel) [inline outplacement_final_report/models/outplacement.py:45]: Missing other_motivation_name?

**Review by nils-vertel — CHANGES_REQUESTED** [review]
Unable to install due to errors in code.
Also please run linter on code to remove various white space and similar

File: outplacement_final_report/models/outplacement.py (hunk 2 above)
Comment (SebastianGieser-Vertel) [reply outplacement_final_report/models/outplacement.py:45]: The xml is what's incorrect.

File: outplacement_final_report/models/outplacement.py (hunk 1 above)
Comment (SebastianGieser-Vertel) [reply outplacement_final_report/models/outplacement.py:39]: I don't see it/get it

**Review by nils-vertel — APPROVED** [review]

**Comment by nils-vertel** [comment]
Merged: https://github.com/vertelab/odoo-outplacement/pull/98

**Pull request merged**